### ISSUE TITLE
Document pooch for dev and users

### ISSUE THREAD

**Issue by sciunto** [scikit-image/scikit-image#4522]
## Description

I think it would be useful to document a bit pooch for dev and users, after merging #3945 

* where the dataset is stored
* how to contribute
* What changes for users

**Comment by jni** [comment]
Yes! Thanks for creating this issue. The contributing guide might be a good place for this. Specifically, if I want to add a documentation example with a large external dataset, what should I do?
